src/app/ToDo/state/todo.httpservice.ts

```diff
@@ -1,21 +1,16 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { from, Observable } from 'rxjs';
-import ToDo from './todo.model';
+import { ToDo } from '.';
 
 @Injectable({
-  providedIn: 'root'
+    providedIn: 'root'
 })
 export class ToDoHttpService {
-  constructor(private httpclient: HttpClient) { }
+    constructor(private httpclient: HttpClient) { }
```

Comment:
`httpClient` :)

Reply:
Is this a criticism or is it ok? hahaha 😁